### ISSUE TITLE
Some of WWDC 2020 Subtitles Download Failed

### ISSUE THREAD

**Issue by kxolecm** [ssamadgh/WWDCsrt#6]
I found the latest WWDCsrt.app (which is **1.6.4 (1)**) can not download subtitles of some **WWDC 2020** Session videos, such as [Explore app clips](https://developer.apple.com/videos/play/wwdc2020/10174/). Please check the screenshots:
<img width="628" alt="截屏2020-08-10 22 30 16" src="https://user-images.githubusercontent.com/5524233/89794203-4b4aff80-db59-11ea-878c-986004e7a277.png">
<img width="628" alt="截屏2020-08-10 22 25 23" src="https://user-images.githubusercontent.com/5524233/89794247-5aca4880-db59-11ea-8365-fa68961b733e.png">

**Comment by ssamadgh** [comment]
Yes, because of changes in 2020 video urls and also in subtitles url WWDC.srt yet not supports WWDC 2020
